DEVELOPMENT.md

```diff
@@ -47,6 +47,29 @@ Spec changes should be approved by a majority of the committers.  Approval can b
 
 No change should be approved until there is documentation for it, supplied in an accompanying PR.
 
+## Draft Features
+
+Where suitable, features will be introduced as draft but OAI approved extensions.
+By introducing new features this way we enable new features to be designed, documented and then implemented by tools that are interested in the feature, without putting the burden of implementation on all tooling.
+If the feature is successfully implemented and there is demonstrable value added by the feature, it will become a candidate for inclusion in a future release of the specification, at which point all tools will be expected to support the feature.
+
+Draft feature extensions are identified by the `x-oas-draft-` prefix and can only be used where existing extensions are permitted.
+This ensures no existing tooling will affected by the introduction of the draft feature.
+If the feature is deemed appropriate for inclusion in the OAS, the `x-oas-draft-` prefix will be removed.
+Tooling that supports draft features should plan for the future removal of the prefix.
+When tooling adds support for a later version of OAS that includes the final implementation of the feature, it MUST not support the use of the draft prefix for that feature.
```

Comment:
@handrews suggested relaxing this:

> I'd say that old x-oas-draft- keywords are still valid, but as with any other x- keyword, support is not reliable. Rather than forbidding them, perhaps implementations SHOULD warn that the feature has graduated?

Was this discussed and rejected for some reason? If not, I think it's a more practical policy, and we should consider adopting it. It will make it easier to migrate OAS documents to new spec versions.